nestor_api/api/api_routes/builds/build_app.py

```diff
@@ -0,0 +1,88 @@
+"""Defines the builds route."""
+
+from http import HTTPStatus
+from threading import Thread
+
+import nestor_api.lib.app as app
+import nestor_api.lib.config as config
+import nestor_api.lib.docker as docker
+import nestor_api.lib.git as git
+import nestor_api.lib.io as io
+from nestor_api.utils.logger import Logger
+
+
+# pylint: disable=broad-except
+def _differed_build(app_name: str):
+    config_dir = None
+    app_dir = None
+
+    try:
+        # Retrieve app's configuration
+        config_dir = config.create_temporary_config_copy()
+        config.change_environment("staging", config_dir)
```

Comment:
Not sure we want to "hardcode" the staging env here, maybe **at least** have this in configuration or retrieve from workflows config in the project.yaml.
Maybe it would be clearer to configure this inside the project directly as an env variable.